src/graphql/checkoutShippingAddressUpdateV2Mutation.graphql

```diff
@@ -3,6 +3,9 @@ mutation checkoutShippingAddressUpdateV2($shippingAddress: MailingAddressInput!,
     userErrors {
       ...UserErrorFragment
     }
+    checkoutUserErrors {
```

Comment:
Does the `checkout-shipping-address-update-v2-fixture` need an empty array added for this field?

Reply:
Good catch